tests/search/utils/elastic_doc_generator.rb

```diff
@@ -53,11 +53,11 @@ def self.write_removes(doc_begin, num_docs, file_name)
   end
 
   def self.write_random_removes(doc_begin, doc_end, num_removes, file_name)
-    generate_random_docs(doc_begin, doc_end, num_removes).write_remove_json(file_name)
+    generate_random_docs(doc_begin, doc_end, num_removes).write_removes_json(file_name)
   end
 
   def self.write_updates(doc_begin, num_docs, file_name, assign_value = 2012)
-    generate_updates(doc_begin, num_docs, assign_value).write_remove_json(file_name)
+    generate_updates(doc_begin, num_docs, assign_value).write_removes_json(file_name)
```

Comment:
generate_updates, but write_removes ... ?

Reply:
Yes, that does not make sense, will fix